Nyron.md

```diff
@@ -0,0 +1,171 @@
+# Week 01
+
+## Instructions
+
+1. Fork this repo
+2. Clone your fork
+3. Fill in your answers by writing in the appropriate area, or placing an 'x' in
+the square brackets (for multiple-choice questions).
+4. Add/Commit/Push your changes to Github.
+5. Open a pull request.
+
+## HTML
+
+### Question #1
+
+What does it mean for an HTML tag to be "semantic"? Give an example of a short snippet of HTML written using semantic tags and non-semantic tags.
+
+<<<<<<< HEAD
```

Comment:
careful with merge conflicts. In this case, looks like you are fine keeping everything in the `HEAD`.